commit 34dbd8acd65523c844385a81239ea0a872750326 (refs/changes/40/4840/3)
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2018-09-17T10:28:26+02:00 (1 year, 1 month ago)
    
    TRITON-772 Deprecate "admin_shadow" from sdc-headnode config
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/scripts/prompt-config.sh b/scripts/prompt-config.sh
index e846ffcb..fa1258f2 100755
--- a/scripts/prompt-config.sh
+++ b/scripts/prompt-config.sh
@@ -1595,15 +1595,6 @@ root_shadow=$(/usr/lib/cryptpass "$root_shadow")
 echo "root_shadow='${root_shadow}'" >>$tmp_config
 echo >>$tmp_config
 
-#
-# Currently we're using the same pw as we use for zones, but we may want
-# to add another prompt for this as a 3rd pw.
-#
-echo "# This is the entry from /etc/shadow for the admin user" >>$tmp_config
-admin_shadow=$(/usr/lib/cryptpass "$zone_admin_pw")
-echo "admin_shadow='${admin_shadow}'" >>$tmp_config
-echo >>$tmp_config
-
 echo "ntp_hosts=$ntp_hosts" >>$tmp_config
 
 echo "compute_node_ntp_hosts=$admin_ip" >>$tmp_config
